__main__.py

```diff
@@ -44,54 +55,87 @@ def execute_instruction(a, b):
 
 
 def create_instruction(a, b):
+    """Creates a full instruction from the contents of two cells
+
+    Args:
+        a (str): The string for the first half of the instruction
+        b (str): The string for the second half of the instruction
+
+    Returns:
+        str: The complete instruction
+    """
     completeinstruction = (a + b)
 
     return completeinstruction
 
 
-# If instruction looks like 1RXY, load register R
-# with the bits found in memory cell XY
 def load_from_cell(instruction):
+    """ If instruction looks like 1RXY
+    load register R with the bits found in memory cell XY
+
+    Args:
+        instruction (str): the instruction being executed, in this case it starts with a 1
+    """
     r = registers[int(instruction[1], 16)]
     xy = instruction[2:]
     xy = cells[int(xy, 16)]
 
     r.setvalue(hex(xy.getvalue()))
 
 
-# If instruction looks like 2RXY, load register R
-# with the bit pattern XY
 def load_with(instruction):
+    """If instruction looks like 2RXY
```

Comment:
Can we move each of the "if instruction looks like ..." to a different line?
Example:
```Load a register R with the bit pattern XY

Instruction should look like (opcode)RXY
...
```

If we want to change the opcodes later, it's easier to just say (opcode) now than changing all of the comments later.